fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryAggregateIndexPlan.java

```diff
@@ -120,7 +124,7 @@ public RecordQueryAggregateIndexPlan(@Nonnull final RecordQueryIndexPlan indexPl
 
     @Nonnull
     @Override
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"unchecked", "resource"})
```

Comment:
This method didn't change. Did something happen that made it necessary to suppress this warning now?